src/jit/gentree.cpp

```diff
@@ -4911,7 +4927,10 @@ unsigned Compiler::gtSetEvalOrder(GenTree* tree)
         case GT_ARR_BOUNDS_CHECK:
 #ifdef FEATURE_SIMD
         case GT_SIMD_CHK:
-#endif                  // FEATURE_SIMD
+#endif // FEATURE_SIMD
+#ifdef FEATURE_HW_INTRINSICS
+        case GT_HWIntrinsic_CHK:
+#endif                  // FEATURE_HW_INTRINSICS
```

Comment:
Would a newline after the `#ifdef` preventing the spacing weirdness on this comment?

Reply:
Good catch! Thanks.